### PR TITLE
Oppref/compress image size

### PR DESCRIPTION
**Pull request by alexcheng068**
-compress images in the photo modals
- default set to compress with option of getting original quality
-  small clean up code

### CODE REVIEW

File: docs/PhotoSelectModal.md [hunk 1]
```diff
@@ -49,7 +50,7 @@ const PhotoSelectModalShowcase = props => {
       <PhotoSelectModal
         visible={modalVisible}
         onHide={() => setModalVisible(false)}
-        onSelectPhoto={uri => setImageURI(uri)}
+        onSelectPhoto={(uri) => setImageURI(uri)}
```
Comment (rachelyellow) [inline docs/PhotoSelectModal.md:53]: onSelectPhoto={setImageURI}

File: package.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "babel-plugin-module-resolver": "^4.0.0",
     "expo-barcode-scanner": "~8.2.1",
-    "expo-camera": "~9.0.0",
+    "expo-camera": "~8.3.1",
```
Comment (sephlao) [inline package.json:25]: why the downgrade?

File: src/components/PhotoSelectModal/PhotoSelectModal.tsx [hunk 3]
```diff
@@ -29,21 +40,19 @@ const PhotoSelectModal: React.FC<PhotoSelectModalProps> = props => {
...
-        visible={props.visible}
-        onHide={props.onHide}
-      />
+      <BottomUpMenu options={options} visible={visible} onHide={onHide} />
       <CameraShootModal
         visible={cameraShootModalVisible}
+        imageCompressed={imageCompressed}
         onHide={() => setCameraShootModalVisible(false)}
-        onSnap={uri => props.onSelectPhoto(uri)}
+        onSnap={(uri) => onSelectPhoto(uri)}
```
Comment (rachelyellow) [inline src/components/PhotoSelectModal/PhotoSelectModal.tsx:48]: onSnap={onSelectPhoto}

File: src/components/PhotoSelectModal/PhotoSelectModal.tsx [hunk 4]
```diff
@@ -29,21 +40,19 @@ const PhotoSelectModal: React.FC<PhotoSelectModalProps> = props => {
...
       />
 
       <ImagePickerModal
         visible={imagePickerVisible}
+        imageCompressed={imageCompressed}
         onHide={() => setImagePickerVisible(false)}
-        onSelect={uri => props.onSelectPhoto(uri)}
+        onSelect={(uri) => onSelectPhoto(uri)}
```
Comment (rachelyellow) [inline src/components/PhotoSelectModal/PhotoSelectModal.tsx:55]: onSelect={onSelectPhoto}

**Review by rachelyellow — CHANGES_REQUESTED** [review]

**Review by anthony-langford — APPROVED** [review]

File: docs/PhotoSelectModal.md (hunk 1 above)
Comment (alexcheng068) [reply docs/PhotoSelectModal.md]: fixed

File: src/components/PhotoSelectModal/PhotoSelectModal.tsx (hunk 3 above)
Comment (alexcheng068) [reply src/components/PhotoSelectModal/PhotoSelectModal.tsx]: fixed

File: src/components/PhotoSelectModal/PhotoSelectModal.tsx (hunk 4 above)
Comment (alexcheng068) [reply src/components/PhotoSelectModal/PhotoSelectModal.tsx]: fixed

File: package.json (hunk 2 above)
Comment (alexcheng068) [reply package.json:25]: there is a version difference between apollo and mobile component, Anthony and I spent 2 hours yesterday trying to resolve it including upgrading apollo but something doesnt work. This is the only work-around for now. Will look into upgrading again later this week

**Review by rachelyellow — APPROVED** [review]

**Review by sephlao — APPROVED** [review]

**Pull request merged**